src/main/java/lotto/domain/AutoLottoGenerator.java

```diff
@@ -19,9 +17,7 @@
     @Override
     public Lotto generateLotto() {
         Collections.shuffle(LOTTO_NUMBER_CARDS);
-        List<LottoNumber> lottoNumbers = LOTTO_NUMBER_CARDS.subList(0, 6);
-        Collections.sort(lottoNumbers);
-        return new Lotto(lottoNumbers);
+        return new Lotto(new TreeSet<>(new ArrayList<>(LOTTO_NUMBER_CARDS.subList(0, 6))));
     }
```

Comment:
```suggestion
        return new Lotto(
            LOTTO_NUMBER_CARDS.stream()
                    .limit(6)
                    .collect(Collectors.toSet())
        );
    }
```

스트림을 이용해 이렇게도 가능할 것 같아요 :)